include/DefaultMQProducer.h

```diff
@@ -28,7 +28,7 @@ namespace rocketmq {
 //<!***************************************************************************
 class ROCKETMQCLIENT_API DefaultMQProducer : public MQProducer {
  public:
-  DefaultMQProducer(const std::string& groupname);
+  DefaultMQProducer(const std::string& groupname, bool Withouttrace = false, void* rpcHook = nullptr);

```

Comment:
notice the name format for funtion variant, such as Withouttrace, please check others